A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A32NX_LocalVarUpdater.js

```diff
@@ -41,4 +41,29 @@ class LocalVarUpdater {
     }
 
     // New selectors go here...
+    _engineOneHydralicsPressure() {
```

Comment:
FYI, variables prefixed with `_` in js mean private/not to be called from outside this file.

Reply:
I'll probably move this to a new file, since it has nothing to do with setting simvars